shenyu-admin/src/main/java/org/apache/shenyu/admin/service/SelectorService.java

```diff
@@ -105,57 +105,63 @@ default int createOrUpdate(SelectorDTO selectorDTO) {
     int updateSelective(SelectorDO selectorDO);
 
     /**
-     * delete selectors.
+     * delete selectors by ids and namespaceId.
      *
-     * @param ids primary key.
+     * @param ids         primary key.
+     * @param namespaceId namespaceId.
      * @return rows int
      */
-    int delete(List<String> ids);
+    int deleteByNamespaceId(List<String> ids, String namespaceId);
 
     /**
-     * find selector by id.
+     * find selector by id and namespaceId.
      *
-     * @param id primary key.
+     * @param id          primary key.
+     * @param namespaceId namespaceId.
      * @return {@linkplain SelectorVO}
      */
-    SelectorVO findById(String id);
+    SelectorVO findByIdAndNamespaceId(String id, String namespaceId);
 
     /**
-     * find selector by name.
+     * find selector by name and namespaceId.
      *
-     * @param name the name
+     * @param name        the name
+     * @param namespaceId namespaceId.
+     *                    Therefore, it is recommended to: {@linkplain SelectorService#findListByNameAndNamespaceId(java.lang.String, java.lang.String)}
      * @return selector do
      * @deprecated sice 2.6.0  Deprecated. By querying under this condition, multiple data are usually obtained.
-     *              Therefore, it is recommended to: {@linkplain SelectorService#findListByName(java.lang.String)}
      */
     @Deprecated
-    SelectorDO findByName(String name);
+    SelectorDO findByNameAndNamespaceId(String name, String namespaceId);
 
     /**
-     * find selector list by name.
+     * find selector list by name and namespaceId.
      *
-     * @param name name
+     * @param name        name
+     * @param namespaceId namespaceId.
      * @return list
      */
-    List<SelectorDO> findListByName(String name);
+    List<SelectorDO> findListByNameAndNamespaceId(String name, String namespaceId);
 
     /**
-     * Find by name and plugin id selector do.
+     * Find selector do by name and pluginName and namespaceId.
      *
-     * @param name       the name
-     * @param pluginName the plugin name
+     * @param name        the name
+     * @param pluginName  the plugin name
+     * @param namespaceId the namespaceId
      * @return the selector do
      */
-    SelectorDO findByNameAndPluginName(String name, String pluginName);
+    SelectorDO findByNameAndPluginNameAndNamespaceId(String name, String pluginName, String namespaceId);
 
     /**
-     * findByNameAndPluginNameForUpdate.
+     * findByNameAndPluginNameAndNamespaceIdForUpdate.
```

Comment:
use normal comment. `find selector entity by selector name, plugin name and namespace id`